Guide.md

```diff
@@ -0,0 +1,116 @@
+# AWS SDK for Rust Developer Guide
+
+This document is a preliminary version of the Developer Guide for the AWS SDK for Rust (the SDK).
+
+This version of the guide is a pared down version of a typical developer guide, with the goal of helping the user quickly and easily download, consume, and use the SDK.
+
+## Getting started 
+
+This section briefly describes:
+
+* How to get an Amazon account
+* How to get your AWS access keys
+* How to install the SDK
+* How to add a dependency to an Amazon or AWS package in Cargo.toml
+
+### Getting an Amazon account
+
+Before you can use the AWS SDK for Rust, you must have an Amazon account. See [How do I create and activate a new Amazon Web Services account?](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account) for details.
+
+### Getting your AWS access keys
+
+Once you have an Amazon account, you need access keys to call any service API using the AWS SDK for Rust. 
+For instructions on creating an access key for an existing IAM user, see [Programmatic access](https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys) in the [IAM User Guide](https://docs.aws.amazon.com/IAM/latest/UserGuide/).
+
+### Installing the SDK
+
+During the early development stage of the SDK, you can install the SDK by unzipping the asset from any of the [releases](https://github.com/awslabs/aws-sdk-rust/releases).
```

Comment:
This step isn't necessary if using Cargo as shown in the `Using packages` section. Cargo will `git clone` the repository at the specified tag and build it from source automatically.

Reply:
I'll remove this paragraph and change the title.